skimage/io/tests/test_collection.py

```diff
@@ -26,14 +26,19 @@ def test_string_sort():
 
 class TestImageCollection(TestCase):
 
-    pattern = [os.path.join(data_dir, pic)
-               for pic in ['camera.png', 'color.png']]
-
-    pattern_matched = [os.path.join(data_dir, pic)
-                       for pic in ['camera.png', 'moon.png']]
-
     def setUp(self):
         reset_plugins()
+        data.fetch('data/color.png')
```

Comment:
This was moved into the setup function so we can deterministically call `data.fetch`. I guess there are other ways of cleaning things up too.